bundles/org.openhab.binding.ws980wifi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,158 @@
+		</channels>
+
+		<properties>                                                                    <!-- static properties, that are the same for all instances -->
+			<property name="vendor">ELV</property>
+			<property name="modelId">WS980WiFi</property>
+		</properties>
+
+		<representation-property>macAddress</representation-property>                   <!-- eindeutiger Identifier fuer jede WS -->
+
+		<config-description>                                                            <!-- konfigurierbare Parameter, indiv. fuer jede WS -->
+			<parameter name="host" type="text" required="true">
+				<label>Hostname</label>
+				<description>The hostname/IP address the device is bound to, e.g. 192.168.0.2</description>
+				<context>network-address</context>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="macAddress" type="text" required="true">
+				<label>MAC Address</label>
+				<description>The unique MAC address of the device, e.g. 00:10:FA:6E:38:4A</description>
+				<context>network-address</context>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="port" type="text" required="true">
+				<label>Data Request Port</label>
+				<default>45000</default>
+				<description>Port number to request data from the device, e.g. 45000</description>
+				<context>network-address</context>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="30">
+				<label>Refresh Interval</label>
+				<default>60</default>
+				<description>
+					Interval the data of the device is polled in sec. (min. 30, default 60)
+				</description>
+				<context>network-address</context>
+				<advanced>false</advanced>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="temperature">                                                     <!-- hier wird der oben referenzierte Type indiv. spezifiziert -->
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Current temperature in degree Celsius</description>
+		<category>Temperature</category>
+		<state pattern="%+.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="rain">
+		<item-type>Number</item-type>
+		<label>Precipitation Rain</label>
+		<description>Rain in 0,1 mm</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.0f"/>
+	</channel-type>
+
+	<channel-type id="lux">
+		<item-type>Number:Illuminance</item-type>
+		<label>Light Level</label>
+		<description>Current Light Level in lux</description>
+		<category>Light</category>
+		<state pattern="%+.0f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="solarRadiation">
+		<item-type>Number:Intensity</item-type>
+		<label>Solar Radiation</label>
+		<description>Solar radiation</description>
+		<category>Ultraviolet</category>
+		<state readOnly="true" pattern="%.2f"/>
+	</channel-type>
+
+	<channel-type id="uvIndex">
+		<item-type>Number</item-type>
```

Comment:
Uv index is created in the code with quantity type `IRRADIANCE`. Thus the correct type and pattern should be used here in the XML.